Saturation/AWPassiveClauseContainer.cpp

```diff
@@ -295,7 +299,9 @@ Clause* AWPassiveClauseContainer::popSelected()
   _size--;
 
   Clause* cl;
-  if (byWeight(_balance)) {
+  bool selByWeight = _randomize ? (Random::getInteger(_ageRatio+_weightRatio) < _weightRatio) : byWeight(_balance);
```

Comment:
A comment would help me here - I expected "fully" (i.e. flip a coin for age or weight) random AWR, but it's actually random-but-respecting-AWR-ratio, I think.

Reply:
Indeed! Comment added..